force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -354,15 +436,23 @@ public with sharing class FlowControlLoopsCollections {
         // add Jane Smith => 60000
         // add Sam Brown => 55000
         // add Alice Johnson => 65000
-
+        employeeSalaries.put('John Doe', 50000);
+        employeeSalaries.put('Jane Smith', 60000);
+        employeeSalaries.put('Sam Brown', 55000);
+        employeeSalaries.put('Alice Johnson', 65000);
```

Comment:
Try using a static variable for employeeSalaries for question 19 and 20 to reduce duplicates since the values are the same